### PR TITLE
AB user profile

### PR DESCRIPTION
**Pull request by aborov**

### CODE REVIEW

File: app/controllers/users_controller.rb
Comment (aizatibraimova) [inline app/controllers/users_controller.rb:1]: Nice implementation! Your show action efficiently retrieves a user based on the username parameter using find_by!

File: app/controllers/comments_controller.rb
Comment (aizatibraimova) [inline app/controllers/comments_controller.rb:1]: Indentation made me confused for a second, but all good. It's just code block got moved by a line.

File: app/views/users/show.html.erb
Comment (aizatibraimova) [inline app/views/users/show.html.erb:1]: Everything is easy to follow.

**Review by aizatibraimova — APPROVED** [review]

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -22,10 +22,11 @@ def edit
   # POST /comments or /comments.json
   def create
     @comment = Comment.new(comment_params)
+    @comment.author = current_user
```
Comment (jniewiel) [inline app/controllers/comments_controller.rb:25]: Nice job assigning the current_user to their respective comments!

File: config/routes.rb
Comment (jniewiel) [inline config/routes.rb:1]: Great job with your routes, and keeping them in order alphabetically! Easy to find.

**Review by jniewiel — APPROVED** [review]
Great job with your pull request!

**Review by JoseMDO — APPROVED** [review]
Great job Aleksei!

**Pull request merged**